src/frontend/src/pages/HomePage/components/TeamWorkPackageDisplay.tsx

```diff
@@ -0,0 +1,64 @@
+import WorkPackageCard from './WorkPackageCard';
+import { wbsPipe } from '../../../utils/pipes';
+import Box from '@mui/material/Box';
+import ScrollablePageBlock from './ScrollablePageBlock';
+import EmptyPageBlockDisplay from './EmptyPageBlockDisplay';
+import CheckCircleOutlineOutlinedIcon from '@mui/icons-material/CheckCircleOutlineOutlined';
+import { AuthenticatedUser } from 'shared';
+
+interface TeamWorkPackageDisplayProps {
+  user: AuthenticatedUser;
+}
+
+const NoTeamWorkPackagesDisplay: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        height: `calc(100vh - 200px)`,
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center'
+      }}
+    >
+      <EmptyPageBlockDisplay
+        icon={<CheckCircleOutlineOutlinedIcon sx={{ fontSize: 128 }} />}
+        heading={"You're team is all caught up!"}
```

Comment:
A more apt title might be "No active work packages" and the message could be "There are no active work packages assigned to your team!"